templates/nextflow-aurora-mysql.yaml

```diff
@@ -60,12 +60,31 @@ Parameters:
     Type: List<String>
     Description: List of admin IAM user and role ARNs (strings)
 
+  DbClusterMapping:
+    Type: String
+    Description: List of admin IAM user and role ARNs (strings)
+    AllowedValues:
+      - mysql5
+      - mysql8
+    Default: mysql5
+
+  KmsKeyAlias:
+    Type: String
+    Description: Alias name of KMS key to use for encryption
+    Default: alias/nextflow-aurora-bootstrap-lambda
+
 Mappings:
   ClusterSettings:
-    mysql:
+    mysql5:
       version: 5.7.mysql_aurora.2.07.1
       engine: aurora-mysql
       family: aurora-mysql5.7
+      enginemode: serverless
+    mysql8:
+      version: 8.0.mysql_aurora.3.04.0
```

Comment:
[Aurora v5.7 maps to Serverless 1 & MySql 2.x and Aurora v8.0 maps to Serverless 2 & MySql 3.x](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-engineversion)

[Serverless 2 maps to MySql 3.x](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Concepts.Aurora_Fea_Regions_DB-eng.Feature.ServerlessV2.html)